ansible/files/epoch.fact

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import yaml
+import json
+
+node_path = "/home/epoch/node"
```

Comment:
I see this is duplicate of https://github.com/aeternity/epoch/blob/e72c0e2128eb82e61b80afa85d159f34cf489c4f/deployment/ansible/deploy.yml#L43

ok.